### PR TITLE
Create GitHub Markdown anchor link id

### PR DESCRIPTION
**Pull request by wildcard**
# New PopClip Extension - Create Github Markdown header links slug id

It Replace spaces between words with **hyphens**, remove **anything that is not a letter, number, space or hyphen** and **downcase**. 

## Example: 

> "The Good, the Bad and the Ugly!" 

=> 

> "the-good-the-bad--the-ugly"

## DEMO - on GitHub Markdown

> create link to section **"String or Dictionary" type** 
the Popclip output is: `string-or-dictionary-type`

image

## Validation

Let's see with it is really the case

expect the link to to the section **"String or Dictionary" type**  be: `#string-or-dictionary-type` 

image

### CODE REVIEW

File: source/GitHubMarkdownAnchorId/Config.plist [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	<array>
+		<dict>
+			<key>After</key>
+			<string>paste-result</string>
+			<key>Script Interpreter</key>
+			<string>/usr/local/bin/bash</string>
+			<key>Shell Script File</key>
+			<string>slugify.sh</string>
+			<key>Title</key>
+			<string>GitHub Anchor Header Link id</string>
```
Comment (invariant) [inline source/GitHubMarkdownAnchorId/Config.plist:15]: I'll rename the extension to somethign a bit snappier to publish it - I'm thinking "Slugify"

File: source/GitHubMarkdownAnchorId/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+wget https://github.com/primer/octicons/raw/master/icons/link.svg
+inkscape -z -w 250 -h 250 link.svg -o link.png
```
Comment (invariant) [inline source/GitHubMarkdownAnchorId/build.sh:4]: Nice!

**Review by invariant — APPROVED** [review]
So, this is the first time I've tried the GitHub review UI. So please bear with me while I'm trying it out.

Looks great. An externsion to create a slug has been on the wishlist for a while. So long I almost though I already had one, but it seems not :)


I had trouble accessing the demo and vaidation gifs. GitHub just showed me some kind of checksum.

**Comment by invariant** [comment]
Merged

**Comment by wildcard** [comment]
I can't see the plugin on https://pilotmoon.com/popclip/extensions/?

**Comment by invariant** [comment]
https://pilotmoon.com/popclip/extensions/page/Slugify

**Pull request closed without merging**